### PR TITLE
Migrate backplane-cli: console

### PR DESCRIPTION
**Pull request by reetika-vyas**
### What type of PR is this?

feature

### What this PR does / Why we need it?

This PR is to migrate backplane console 

### Which Jira/Github issue(s) does this PR fix?

[OSD-14685](https://issues.redhat.com/browse/OSD-14685)

### Pre-checks (if applicable)

- [ Yes] Ran unit tests locally
- [ Yes ] Validated the changes in a cluster
- [NO  ] Included documentation changes with PR

### CODE REVIEW

File: cmd/ocm-backplane/root.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/managedJob"
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/script"
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/status"
+	"github.com/openshift/backplane-cli/cmd/ocm-backplane/console"
```
Comment (feichashao) [inline cmd/ocm-backplane/root.go:29]: Suggest moving this upper after "github.com/openshift/backplane-cli/cmd/ocm-backplane/cloud" to follow the alphabet order.

File: cmd/ocm-backplane/root.go [hunk 2]
```diff
@@ -21,11 +21,12 @@ import (
 	"strings"
 
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/cloud"
+	"github.com/openshift/backplane-cli/cmd/ocm-backplane/console"
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/login"
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/logout"
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/managedJob"
 	"github.com/openshift/backplane-cli/cmd/ocm-backplane/script"
-	"github.com/openshift/backplane-cli/cmd/ocm-backplane/status"
+	"github.com/openshift/backplane-cli/cmd/ocm-backplane/status"
```
Comment (feichashao) [inline cmd/ocm-backplane/root.go:29]: Remove the extra tab in the end.

File: cmd/ocm-backplane/root.go [hunk 3]
```diff
@@ -95,4 +96,6 @@ func init() {
 	rootCmd.AddCommand(version.VersionCmd)
 	rootCmd.AddCommand(testJob.NewTestJobCommand())
 	rootCmd.AddCommand(managedJob.NewManagedJobCmd())
+	rootCmd.AddCommand(console.ConsoleCmd)
+
```
Comment (supreeth7) [inline cmd/ocm-backplane/root.go:100]:
```suggestion
```

File: cmd/ocm-backplane/console/console_test.go [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+		})
+	})
+
+	// This test verifies that the console container is still started the same way after issuing a
+	// 'oc project <namespace id>' command.
+	//
+	// In particular this test checks that the name of container started by the 'ocm backplane console'
+	// command is based on the cluster id and not on the supposed cluster name extracted from kube config.
+	//
+	// Indeed 'oc' client is actually connected to the hive cluster which proxy commands to the targeted
```
Comment (supreeth7) [inline cmd/ocm-backplane/console/console_test.go:157]: We do not want any code/comments that mentions Hive
```suggestion
```

File: cmd/ocm-backplane/console/console_test.go [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+	// OSD cluster.
+	// Issuing a 'oc project <namespace id>' will create a new context with a new cluster in kube config...
+	// but the name of the newly created cluster config will be based on the hive cluster URL:
+	// - Which does not contain any bit of information concerning the OSD cluster name.
+	// - Which contains ':' char which is an invalid char in a container name.
```
Comment (supreeth7) [inline cmd/ocm-backplane/console/console_test.go:162]:
```suggestion
```

File: cmd/ocm-backplane/console/console_test.go (hunk 4 above)
Comment (supreeth7) [inline cmd/ocm-backplane/console/console_test.go:157]:
```suggestion
```

**Pull request merged**